app/src/main/java/com/vero/tasky/agenda/data/remote/network/request/SyncAgendaRequest.kt

```diff
@@ -4,7 +4,7 @@ import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
 data class SyncAgendaRequest (
-    val deletedEventsIds : List<String>,
-    val deletedTasksIds: List<String>,
-    val deletedReminderIds: List<String>,
+    val deletedEventIds : Array<String>,
+    val deletedTaskIds: Array<String>,
+    val deletedReminderIds: Array<String>,
```

Comment:
Why arrays instead of List?

Personally I always use Lists. I find them way easier to use than Arrays